packages/@ama-sdk/schematics/schematics/typescript/shell/index.spec.ts

```diff
@@ -78,7 +78,7 @@ describe('Typescript Shell Generator', () => {
       skipInstall: true,
       packageManager: 'npm'
     }, Tree.empty());
-  }, 15000);
+  }, 25000);
```

Comment:
In my PR about reworking the unit tests, I've increased the global testTimeout to 30s https://github.com/AmadeusITGroup/otter/pull/1276/files#diff-cacec94ff84332c24eff579dc017461bb1cd2caf48ef3a408a70ecbcff09738eR74

Reply:
then i flag it duplicate of #1276